### PR TITLE
turned *Data & value into *Schema

### PR DESCRIPTION
**Pull request by vcharpenay**

### CODE REVIEW

File: index.html.template [hunk 1]
```diff
@@ -155,7 +155,7 @@ Example 1 shows a simple TD instance that reflects WoT's Property, Action, Event
         {
             "@type": ["Property"],
             "name": "status",
-            "outputData": {"type": "string"},
+            "outputSchema": {"type": "string"},
```
Comment (zolkis) [inline index.html.template]: schema

File: index.html.template [hunk 2]
```diff
@@ -174,7 +174,7 @@ Example 1 shows a simple TD instance that reflects WoT's Property, Action, Event
         {
             "@type": ["Event"],
             "name": "overheating",
-            "outputData": {"type": "string"},
+            "outputSchema": {"type": "string"},
```
Comment (zolkis) [inline index.html.template]: schema

**Comment by ToruKawaguchi** [comment]
Looks there is some mistakes on the SVG diagram. Property has outputSchema, Action has inputSchema and schema. ,

**Comment by vcharpenay** [comment]
Thanks @ToruKawaguchi! Fixed in PR #96.

**Pull request merged**